Sorts/test/QuickSort.test.ts

```diff
@@ -0,0 +1,26 @@
+import { QuickSort } from "../QuickSort";
+
+/**
+ * Space complexity - O(nlogn)
```

Comment:
should be a comment on the quick sort func

Reply:
Shifted the comment to QuickSort.ts